hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmBucketInfo.java

```diff
@@ -188,6 +218,30 @@ public BucketEncryptionKeyInfo getEncryptionKeyInfo() {
     return bekInfo;
   }
 
+  /**
+   * Set the Object ID. If this value is already set then this function throws.
+   * There is a reason why we cannot use the final here. The OmVolumeArgs is
+   * deserialized from the protobuf in many places in code. We need to set
+   * this object ID, after it is deserialized.
+   *
+   * @param obId - long
+   */
+  public void setObjectID(long obId) {
+    if(this.objectID != 0) {
```

Comment:
Do we have guarranty that ratisTransactionIndex starts from 1?

Becoz lets take a scenario, 0 - createBucket 1 - setBucketProperty, then when setBucketProperty calls I can still call setObjectID and update objectID.

Reply:
RatisTransactionIndex starts from 0 but the first index is always taken up by the conf change for leader election. I will confirm this.